src/config/connection.js

```diff
@@ -0,0 +1,32 @@
+const Sequelize = require("sequelize");
+
+// const dbName = process.env.DB_NAME;
+// const dbUser = process.env.DB_USER;
+// const dbPassword = process.env.DB_PASSWORD;
+// const dbHost = process.env.DB_HOST;
+
+// const dbOptions = {
+//   host: dbHost,
+//   dialect: 'mysql',
+//   port: 3306,
+//   logging: false,
+// };
+
+// const sequelize = new Sequelize(dbName, dbUser, dbPassword, dbOptions);
+
+const sequelize = new Sequelize(
+  // Database name
+  "ecommerce_db",
+  // User
+  "root",
+  // Password
+  "Password123!!",
```

Comment:
Now that you know about the .env package, please don't push your passwords to github! This will cause lots of issues with your projects if you do this with any API keys as it will be pulled as a security breach. The reason the commented out code above wasn't working is because you need to require the dotenv package at the top.